data_extrator.py

```diff
@@ -22,14 +18,26 @@ def exponential_filter(time_step):
         return filtered_data[-parameters_per_slice:]
 
 
-    parameters = np.zeros((slice_count, parameters_per_slice))
-    for i in range(slice_count):
-        # Calculate last point of slice
-        time_step = time_step_0 - i*slice_shift
-        # Calculate last 6 points of filtered values
-        parameters[i] = exponential_filter(time_step)
+    def exponential_filter_parameters(data, time_step_0, slice_count=5, slice_overlap=50, alpha_value = 0.8, slice_size = 100, parameters_per_slice = 6, plot_results = False):
```

Comment:
Niektóre parametry są stałe w trakcie eksperymentu i lepiej je wrzucić do konstruktora klasy:
- data
- slice_count
- slice_overlap
- slice_size
- parameters_per_slice (nazwa trochę zbyt ogólny, bo to dotyczy tylko filtru wykładniczego, ale na razie chyba może być)

Co do alpha_value, to każdy slice może mieć inną wartość, więc to powinna być lista zawierająca alpha_value dla każdego slice'a.